src/templates/web/base.html

```diff
@@ -0,0 +1,241 @@
+                    <p>
+                        <a href="catalogo.html" class="text-white" style="text-decoration: none;">Inicio</a>
+                    </p>
+                    <p>
+                        <a href="#" class="text-white" style="text-decoration: none;">Quienes somos</a>
+                    </p>
+                    <p>
+                        <a href="#" class="text-white" style="text-decoration: none;">Catálogo</a>
+                    </p>
+                    <p>
+                        <a href="#" class="text-white" style="text-decoration: none;">Tienda</a>
+                    </p>
+                    <p>
+                        <a href="#" class="text-white" style="text-decoration: none;">Contacto</a>
+                    </p>
+
+                </div>
+
+
+                <div class="col-md-4 col-lg-3 col-xl-3 mx-auto mt-3">
+                    <div class=" font-weight-bold  ">
+                        <h5 class="text-warning text-uppercase mb-4 text-md-center" style="font-size: 24px;">
+                            ¡Suscríbete!</h5>
+                            <hr class="mb-4 ">
+                            <p>
+                                <i class="text-white "></i> Recibe todas las ofertas directo a tu correo
+                            </p>
+                            <input type="email" class="footer__input" placeholder="correo@ejemplo.com">
+                    </div>
+                </div>
+
+            </div>
+
+            <hr class="mb-4 ">
+            <div class="row align-items-center">
+                <div class="col-md-7 col-lg-8">© 2022 Arte Manga. Todos los derechos reservados
+
+                    <img src="{% static 'img/manga.png' %}" width="150px" height="100px">
+
+                </div>
+
+
+
+                <div class="col-md-5 col-lg-4">
+                    <div class="text-center text-md-right">
+
+                        <ul class="list-unstyled list-inline">
+                            <li class="list-inline-item">
+                                <a href="#" class="btn-floating btn-sm text-white" style="font-size: 24px;">
+                                    <i class="fab fa-facebook"></i></a>
+                            </li>
+                            <li class="list-inline-item">
+                                <a href="#" class="btn-floating btn-sm text-white" style="font-size: 24px;">
+                                    <i class="fab fa-twitter"></i></a>
+                            </li>
+                            <li class="list-inline-item">
+                                <a href="#" class="btn-floating btn-sm text-white" style="font-size: 24px;">
+                                    <i class="fab fa-instagram"></i></a>
+                            </li>
+                        </ul>
+
+                    </div>
+                </div>
+            </div>
+
+    </footer>
+
+    {% block extra_javascript %}
+    {% endblock %}
+
+</body>
+
+</html>
```

Comment:
```suggestion
</html>

```

ese circulito rojo significa que falta una línea vacía al final, es mejor agregarla para evitar tonteras con linux (el server de prueba corre en linux y el de producción también correría en linux)